### DISCUSSION TITLE
K1SE

### DISCUSSION THREAD

**Discussion by milczas** [Guilouz/Creality-Helper-Script-Wiki#731] [Issues]
Is it possible? Maybe someone has already modified the K1SE printer to K1C. I already have new sides and doors as in K1C, but the printer software does not see additional fans. I would like to install them. Maybe there is a procedure to change the software borrowed from K1C

**Comment by pbroggero** [comment]
Software solution is here (sorry is in french)
https://github.com/Guilouz/Creality-Helper-Script-Wiki/discussions/807
